blivet/devices/lvm.py

```diff
@@ -1808,9 +1809,11 @@ def exists(self):
 
     @property
     def stats(self):
-        if not self._exists:
+        # to get the stats we need the cached LV to exist and be activated
```

Comment:
This would be easier to read if it was flipped, if self._exists and self._cached_lv.status: return LVMCacheStats...  else: return None